metricbeat/module/kibana/kibana.go

```diff
@@ -159,5 +159,6 @@ func fetchPath(http *helper.HTTP, currentPath, newPath string) ([]byte, error) {
 
 	// Http helper includes the HostData with username and password
 	http.SetURI(u.String())
+	http.SetHeader("Authorization", "ApiKey "+apiKey)
```

Comment:
[Question]
Should this header be always set or only when `apiKey` is present (not an empty string)?